engine/storage/volume/src/org/apache/cloudstack/storage/datastore/PrimaryDataStoreImpl.java

```diff
@@ -239,10 +241,34 @@ public boolean isManaged() {
         return pdsv.isManaged();
     }
 
+    private boolean canCloneVolume() {
+        return new Boolean(getDriver().getCapabilities().get(DataStoreCapabilities.CAN_CREATE_VOLUME_FROM_VOLUME.toString()));
```

Comment:
``new Boolean`` skips the constant pool -- placing unnecessary pressure on the garbage collector.  Please use ``Boolean.valueOf`` instead.